command-based/subsystems/singlemotor.py

```diff
@@ -2,19 +2,18 @@
 from wpilib.command.subsystem import Subsystem
 
 from commands.followjoystick import FollowJoystick
-import robotmap
 
 class SingleMotor(Subsystem):
     '''
-    This example subsystem controls a single CAN Talon SRX in PercentVBus mode.
+    This example subsystem controls a single Talon in PercentVBus mode.
     '''
 
     def __init__(self):
         '''Instantiates the motor object.'''
 
         super().__init__('SingleMotor')
 
-        self.motor = wpilib.Talon(robotmap.singlemotor.motorID)
+        self.motor = wpilib.Talon(1)
```

Comment:
Here's an opinionated suggestion: what about passing the port number in the constructor (so instantiating this class as `SingleMotor(1)` in robot.py)?

Reply:
This is just an example subsystem. A real subsystem would likely have many ports related to it, so passing them all in the constructor would get a bit overwhelming.

This commit does allow edits, so you may make that change if you want, but we aren't planning to put more effort into it.